acquisitions/src/main/resources/domain/mod-invoice/invoice-junit.feature

```diff
@@ -0,0 +1,28 @@
+Feature: mod-invoice integration tests
+
+  Background:
+    * def testAdmin = {tenant: '#(testTenant)', name: 'test-admin', password: 'admin'}
```

Comment:
remove this
 def testAdmin = {tenant: '#(testTenant)', name: 'test-admin', password: 'admin'}
    * def testUser = {tenant: '#(testTenant)', name: 'test-user', password: 'test'}

Reply:
Done

Reply:
Will remove it, thank you.